atom/browser/api/atom_api_app.cc

```diff
@@ -1112,30 +1112,11 @@ std::vector<mate::Dictionary> App::GetAppMetrics(v8::Isolate* isolate) {
 
   for (const auto& process_metric : app_metrics_) {
     mate::Dictionary pid_dict = mate::Dictionary::CreateEmpty(isolate);
-    mate::Dictionary memory_dict = mate::Dictionary::CreateEmpty(isolate);
     mate::Dictionary cpu_dict = mate::Dictionary::CreateEmpty(isolate);
 
     pid_dict.SetHidden("simple", true);
-    memory_dict.SetHidden("simple", true);
     cpu_dict.SetHidden("simple", true);
 
-    memory_dict.Set(
-        "workingSetSize",
-        static_cast<double>(
-            process_metric.second->metrics->GetWorkingSetSize() >> 10));
-    memory_dict.Set(
-        "peakWorkingSetSize",
-        static_cast<double>(
-            process_metric.second->metrics->GetPeakWorkingSetSize() >> 10));
-
-    size_t private_bytes, shared_bytes;
-    if (process_metric.second->metrics->GetMemoryBytes(&private_bytes,
-                                                       &shared_bytes)) {
-      memory_dict.Set("privateBytes", static_cast<double>(private_bytes >> 10));
-      memory_dict.Set("sharedBytes", static_cast<double>(shared_bytes >> 10));
-    }
-
-    pid_dict.Set("memory", memory_dict);
```

Comment:
this is a breaking change, is it documented somewhere? (Also, as per previous discussions about removing this API, I think it's important that we replace it with something at least as good.)

cc @nitsakh

Reply:
>  is it documented somewhere
Yeap, it's already in [the breaking changes doc](https://github.com/electron/electron/blob/master/docs/api/breaking-changes.md).

Reply:
> t's important that we replace it with something at least as good

We have a task for this: https://github.com/electron/electron/issues/13447